scripts/release_ppa.sh

```diff
@@ -51,17 +54,43 @@ is_release() {
     [[ "${branch}" =~ ^v[0-9]+(\.[0-9]+)*$ ]]
 }
 
-keyid=379F4801D622CDCF
-email=builds@ethereum.org
+# source keyid and email from .release_ppa_auth
+if [ -e .release_ppa_auth ]
+then
+    # shellcheck source=/dev/null
+    source .release_ppa_auth
+fi
+
+if [ -z "$LAUNCHPAD_KEYID" ] || [ -z "$LAUNCHPAD_EMAIL" ]
+then
+    echo "Error: Couldn't find variables \$LAUNCHPAD_KEYID or \$LAUNCHPAD_EMAIL in sourced file .release_ppa_auth (check top comment in $0 for more information)."
```

Comment:
This should go to stderr:
```suggestion
    >&2 echo "Error: Couldn't find variables \$LAUNCHPAD_KEYID or \$LAUNCHPAD_EMAIL in sourced file .release_ppa_auth (check top comment in $0 for more information)."
```
You could also import the `fail` function from `scripts/common.sh` and make this more concise:
```bash
[[ $LAUNCHPAD_KEYID != "" && $LAUNCHPAD_EMAIL != "" ]] || \
    fail "Error: Couldn't find variables \$LAUNCHPAD_KEYID or \$LAUNCHPAD_EMAIL in sourced file .release_ppa_auth (check top comment in $0 for more information)."
```